More items in Pipe dispenser and more

Laboredih123 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
Adds the Dual port vent, the pressure tank, the multi z pipe, and the large versions of vents, scrubbers, and dual port vents to the RPD.
Adds to pipe-item.dmi to not have the dispenser showing a simple pipe
Makes tanks destructible so you don't have to hack away at a tank every time you misplace one
Minor add of RELEASING and SIPHONING to the DPVENT

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.
Also put closed issues under this tag, if any. Format is as follows(must be lowercase):
closes #123456789
-->
More variety and stuff
Cleanliness


## Changelog
:cl:
add: More things in the Pipe Dispenser, check em out
code: Added a define for cleanliness in dpvents
add: Added some sprites to pipe-item
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Laboredih123 committed:
Port from original PR

KubeRoot commented on code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm:
```diff
@@ -267,6 +267,18 @@
+	name = "large scrubber"
+	idle_power_usage = 25
+	active_power_usage = 150
+	volume_rate = 1000
```
5x the volume rate at less than 3x the active power usage? I'm iffy on the idea of a large scrubber in the first place, but if it *is* included, I think it should have much higher power usage to compensate for the added utility.

Laboredih123 replied:
Values are now 6x for the large scrubber

KubeRoot commented on code/game/objects/items/RPD.dm:
```diff
@@ -22,22 +22,28 @@ GLOBAL_LIST_INIT(atmos_pipe_recipes, list(
+		new /datum/pipe_info/pipe("Dual-port Air Vent",			/obj/machinery/atmospherics/components/binary/dp_vent_pump),
+		new /datum/pipe_info/pipe("Large Dual-port Air Vent",	/obj/machinery/atmospherics/components/binary/dp_vent_pump/high_volume),
+		new /datum/pipe_info/pipe("Pressure Tank",				/obj/machinery/atmospherics/components/unary/tank),
+
```
Random empty line here

Laboredih123 replied:
Fixed

KubeRoot commented on code/game/objects/items/RPD.dm:
```diff
@@ -22,22 +22,28 @@ GLOBAL_LIST_INIT(atmos_pipe_recipes, list(
 		new /datum/pipe_info/meter("Meter"),
-		new /datum/pipe_info/pipe("Gas Filter",			/obj/machinery/atmospherics/components/trinary/filter),
-		new /datum/pipe_info/pipe("Gas Mixer",			/obj/machinery/atmospherics/components/trinary/mixer),
-		new /datum/pipe_info/pipe("Pressure Valve",		/obj/machinery/atmospherics/components/binary/pressure_valve),
-		new /datum/pipe_info/pipe("Temperature Gate",	/obj/machinery/atmospherics/components/binary/temperature_gate),
-		new /datum/pipe_info/pipe("Temperature Pump",	/obj/machinery/atmospherics/components/binary/temperature_pump),
```
You appear to be removing the pressure valve, temperature gate and temperature pump from the RPD.

Laboredih123 replied:
Fixed

KubeRoot reviewed (changes requested):
I'll want to test this in practice at some point, might also want to review potential balance implications from adding buildable large things.

I also think the pressure tank should maybe require material, for example make it so the RPD builds a frame that needs to have plasteel applied to finish building it.

Laboredih123 committed:
Fixes the RPD items and comments out multi z pipes
Removes unnecessary new in the dp vent
Greatly amps up the power usage for large scrubbers from the original ported version
Returns ..() in tank/Destroy()

Laboredih123 commented on code/modules/atmospherics/machinery/components/binary_devices/dp_vent_pump.dm:
```diff
@@ -38,6 +43,12 @@
+
+/obj/machinery/atmospherics/components/binary/dp_vent_pump/New()
+	..()
+
```
Removed

Laboredih123 commented on code/modules/atmospherics/machinery/components/unary_devices/tank.dm:
```diff
@@ -24,6 +26,54 @@
+		var/turf/T = loc
+		T.assume_air(air_contents)
+		air_update_turf()
+	..()
```
Done

Laboredih123 commented on code/modules/atmospherics/machinery/components/unary_devices/tank.dm:
```diff
@@ -24,6 +26,54 @@
+	..()
+
+/obj/machinery/atmospherics/components/unary/tank/welder_act(mob/living/user, obj/item/I)
+	..()
```
Removed as it served no purpose here

Laboredih123 committed:
Removus tank from RPD

Laboredih123 commented:
Removed the pressure tank from the RPD, plan to make it a buildable machine.

Laboredih123 commented:
its not meant to close...
